### PR TITLE
doc: added changelog since latest version.

### PR DESCRIPTION
**Pull request by meathill**
and change test versions to 0.27

### CODE REVIEW

File: lib/Test/Nginx/LWP.pm [hunk 1]
```diff
@@ -495,11 +495,11 @@ Redistributions in binary form must reproduce the above copyright notice, this l
...
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```
Comment (agentzh) [inline lib/Test/Nginx/LWP.pm:502]: What is changed on this line?

File: lib/Test/Nginx/Socket.pm [hunk 2]
```diff
@@ -3662,7 +3662,7 @@ Below is an example from ngx_headers_more module's test suite:
     --- response_headers
     ! X-Foo
     --- response_body
-    x-foo: 
+    x-foo:
```
Comment (agentzh) [inline lib/Test/Nginx/Socket.pm:3665]: Don't do this. This line trailing space is intentional and required.

File: lib/Test/Nginx/LWP.pm (hunk 1 above)
Comment (meathill) [reply lib/Test/Nginx/LWP.pm:502]: The trailing space in this line was auto removed by my editor.

File: lib/Test/Nginx/Socket.pm (hunk 2 above)
Comment (meathill) [reply lib/Test/Nginx/Socket.pm]: Added back.

**Pull request closed without merging**